src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+    position: relative;
+    .intro {
+      max-width: 50%;
+      width: auto;
+      height: auto;
+      position: relative;
+      display: inline-block;
+      margin: 0 20px 20px;
+    }
+    .experience {
+      max-width: 33.3%;
+      width: auto;
+      height: auto;
+      position: relative;
+      display: inline-block;
+      margin: 0 20px;
+      a {
+        text-decoration: none;
+        color: $primary_color;
+      }
+    }
+    .education {
+      max-width: 33.3%;
+      width: auto;
+      height: auto;
+      position: relative;
+      display: inline-block;
+      margin: 0 20px;
+    }
+    .hobbies {
+      max-width: 15%;
+      width: 100px;
+      height: auto;
+      position: relative;
+      display: inline-block;
+      margin: 15px 20px 0;
+      cursor: pointer;
+      i {
+        font-size: 50px;
+        text-align: center;
+        margin: 0 20px;
+      }
+      p {
+        font-size: 20px;
+        text-align: center;
+      }
+    }
+  }
+}
+
+.dot-sections {
+  text-align: center;
+  margin-top: 20px;
+  .dot {
+    cursor: pointer;
+    height: 15px;
+    width: 15px;
+    margin: 0 2px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+  }
+  .dot:hover {
+    background-color: $btn_hover_color;
+  }
+  .active {
+    background: $btn_hover_color;
+  }
+}
+
+//fade animation
+.fade {
+  -webkit-animation-name: fade;
+  -webkit-animation-duration: 1.5s;
+  animation-name: fade;
+  animation-duration: 1.5s;
+}
+
+@-webkit-keyframes fade {
+  from {
+    opacity: 0.4;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes fade {
+  from {
+    opacity: 0.4;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.skills-section {
+  max-width: 1000px;
+  position: relative;
+  margin: auto;
+  color: $primary_color;
+  padding: 0 20px;
+  h2 {
+    color: $secondary_color;
+  }
+  .skill {
+    margin: 0 20px;
+    max-width: 33.3%;
+    position: relative;
+    width: auto;
+    height: auto;
+    display: inline-block;
+  }
+}
+
+.contacts-section {
+  max-width: 1000px;
+  margin: auto;
+  color: $primary_color;
+  padding: 0 20px;
+  position: relative;
+  display: flex;
+  margin-bottom: 30px;
+  .id-section {
+    max-width: 50%;
+    position: relative;
+    width: 50%;
+    height: auto;
+    display: inline-block;
+  }
+  .border {
+    width: 100px;
+    border: 3px solid $btn_hover_color;
+    margin-bottom: 20px;
+  }
+  .contact_details {
+    font-weight: bold;
+    td:nth-child(1) {
+      width: 100px;
+      color: $secondary_color;
+    }
+    td:nth-child(2) {
+      width: 200px;
+      color: $secondary_color;
+    }
+    a {
+      text-decoration: none;
+      color: $secondary_color;
+    }
+  }
+
+  .contact-me-section {
+    max-width: 50%;
+    position: relative;
+    width: 50%;
+    height: auto;
+    display: inline-block;
+    form {
+      text-align: center;
+      input {
+        padding: 10px 10px;
+      }
+      input[type='text']:nth-child(1) {
```

Comment:
In the HTML section, I mentioned you are missing all `name` attribute.
That's why you have to rely on `nth-child()` not `input[name=email]`....
This is considered not professional since you style if based on element order.